gulp/tasks/pldoc_scripts.js

```diff
@@ -17,3 +17,14 @@ gulp.task('pldoc_scripts', ['pldoc_scripts-lint'], function() {
         .pipe(gulp.dest(config.pldoc_dest))
         .pipe(browserSync.reload({stream:true}))
 });
+
+gulp.task('pldoc_scripts', ['pldoc_scripts-lint'], function() {
+    return gulp.src([
+        // setup sequence for files that are loaded later with RequireJS
+        './pattern-library/js/*'
+    ])
+    .pipe(uglify())
+    .pipe(gulp.dest(config.pldoc_local)) // move for just browsercync + local preview
+    .pipe(gulp.dest(config.pldoc_dest))
+    .pipe(browserSync.reload({ stream: true }))
+});
```

Comment:
See [my note above on using a config variable](https://github.com/edx/ux-pattern-library/pull/167/files#r39582587). I think its doable and not too headache inducing.